### PR TITLE
feat(dapi): serve even if tenderdash is not connected

### PR DESCRIPTION
**Pull request by shumkov**
## Issue being fixed or feature implemented

During activation or any other reason when Tenderdash RPC is not responding we still want DAPI to respond to RPC requests which aren't required Tenderdash to operate.

## What was done?

- Do not block API to start if tenderdash is not connected
- Respond with Unavilable gRPC error from endpoints that require Tenderdash to be responding
- Removed two unused endpoints

## How Has This Been Tested?

Tested locally

## Breaking Changes

None

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have added "!" to the title and described breaking changes in the corresponding section if my code contains any
- [x] I have made corresponding changes to the documentation if needed

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: packages/dapi/scripts/api.js [hunk 1]
```diff
@@ -107,17 +107,16 @@ async function main() {
...
-  tenderDashWsClient.on('disconnect', () => {
-    tenderdashLogger.fatal('Disconnected from Tenderdash... exiting');
+  const blockchainListener = new BlockchainListener(tenderDashWsClient);
+
+  tenderDashWsClient.connect().then(() => {
+    blockchainListener.start();
+  }).catch((e) => {
+    tenderdashLogger.fatal(`Not able to connect to tenderdash: ${e}`);
 
     process.exit(1);
```
Comment (pshenmic) [inline packages/dapi/scripts/api.js:117]: We still exit in case cannot connect to tenderdash?

Comment (shumkov) [reply packages/dapi/scripts/api.js:117]: It's not gonna happen with current setting `maxRetry: -1` but I would like to have an error handling here anyways. For example if we have some syntax error or other problem.

Comment (pshenmic) [reply packages/dapi/scripts/api.js:117]: So users still not able to query DAPI until tenderdash actually starts?

**Review by pshenmic — APPROVED** [review]
utACK

File: packages/dapi/scripts/api.js [hunk 2]
```diff
@@ -107,17 +107,16 @@ async function main() {
...
   });
 
-  tenderDashWsClient.on('disconnect', () => {
-    tenderdashLogger.fatal('Disconnected from Tenderdash... exiting');
+  const blockchainListener = new BlockchainListener(tenderDashWsClient);
+
+  tenderDashWsClient.connect().then(() => {
+    blockchainListener.start();
+  }).catch((e) => {
+    tenderdashLogger.fatal(`Not able to connect to tenderdash: ${e}`);
```
Comment (lklimek) [inline packages/dapi/scripts/api.js:115]: Double-check if `process.exit(1)` below is correct, if I understand correctly, we intended to not exit in this case.

File: packages/dapi/scripts/api.js [hunk 3]
```diff
@@ -107,17 +107,16 @@ async function main() {
     tenderdashLogger.warn(`Connection closed: ${error.code}`);
   });
 
-  tenderDashWsClient.on('disconnect', () => {
```
Comment (lklimek) [inline packages/dapi/scripts/api.js:110]: does it mean we don't handle disconnect event at all anymore? I think we should at least log this.

File: packages/dapi/scripts/api.js (hunk 2 above)
Comment (shumkov) [reply packages/dapi/scripts/api.js:115]: Yes, it will happen it case of the bug in connect function or maRetires != -1 (which is no the case)

**Pull request merged**